include/itkMaxPhaseCorrelationOptimizer.h

```diff
@@ -46,7 +43,7 @@ namespace itk
  *
  */
 template < typename TRegistrationMethod >
-class ITK_EXPORT MaxPhaseCorrelationOptimizer :
```

Comment:
Shouldn't this be `ITK_TEMPLATE_EXPORT`? The other cases too.

Reply:
Yes, good catch.